android-components/components/browser/engine-system/src/main/java/mozilla/components/browser/engine/system/SystemEngineSession.kt

```diff
@@ -492,6 +492,17 @@ class SystemEngineSession(
         throw UnsupportedOperationException("Sending impression attribution event is not available in this engine")
     }
 
+    /**
+     * See [EngineSession.sendPlacementAttributionEvent]
+     */
+    override fun sendPlacementAttributionEvent(
```

Comment:
We should mark this function with @Throws annotation for code inspection.

Reply:
I left this as is for now. If you'd like to file a bug to add this for all the functions, this can be worked on.